Src/BootCamp.Chapter/BMISLeslie

```diff
@@ -0,0 +1,38 @@
+using System;
+
+namespace BMISLeslie
+{
+    class Program
+    {
+        private static void Main(string[] args)
+        {
+            //Declare variables
+            string name;
+            string surname;
+            int age; 
+            decimal height, weight;
+            decimal bmi;
+            string printOut;
+
+            //Console Entries and add value to variables
+            Console.WriteLine("Please enter your first name:");
+            name = (Console.ReadLine());
+            Console.WriteLine("Please Enter your surname:");
+            surname = (Console.ReadLine());
+            Console.WriteLine("Please enter your age:");
+            age = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("Please enter your height in cm:");
+            height = Convert.ToDecimal(Console.ReadLine());
+            Console.WriteLine("Please enter your weight in kg:");
+            weight = Convert.ToDecimal(Console.ReadLine());
+
+            //Work out BMI
+            bmi = weight / ((height / 100) * (height / 100));
+            
+            //Printout
+            printOut = name + " " + surname + " is " + age + " years old, his weight is " + weight + "kg and his height is " + height + "cms. His BMI is " + Math.Round(bmi, 2) + ".";
```

Comment:
C# has cool feature for this: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated

Take a look and try to use in the future 😊